lib/protocol/http/header/etags.rb

```diff
@@ -2,21 +2,42 @@
 
 # Released under the MIT License.
 # Copyright, 2020-2023, by Samuel Williams.
+# Copyright, 2023, by thomas morgan.
```

Comment:
Do you mind making sure your git user name has appropriate capitalisation? The name for copyright is taken from the history.

Reply:
Not really up for changing my standard commit config, but it looks like I can update `.mailmap` for that (thanks for supporting such an option!) I'll get that done and pushed here shortly.

Reply:
You can also just change it per project if that is easier. Either option is fine.